hazelcast/src/test/java/com/hazelcast/multimap/MultiMapTest.java

```diff
@@ -104,6 +114,184 @@ public void testMultiMapPutGetRemove() {
         assertEquals(0, multiMap.size());
     }
 
+    protected <K, V> EntryListener<K, V> putAllEntryListenerBuilder(Consumer<EntryEvent<K, V>> f) {
+        return new EntryAdapter<K, V>() {
+            public void entryAdded(EntryEvent<K, V> event) {
+                f.accept(event);
+            }
+
+            public void entryRemoved(EntryEvent<K, V> event) {
+                System.out.println("entryRemoved " + event);
```

Comment:
as soon as you no longer need these `System.out`s (probably inserted for debugging?), please remove them.